game-core/src/main/java/games/strategy/triplea/ui/PoliticsPanel.java

```diff
@@ -221,8 +222,9 @@ private JPanel getOtherPlayerFlags(final PoliticalActionAttachment paa) {
     return panel;
   }
 
-  private static String getActionButtonText(final PoliticalActionAttachment paa) {
-    final String costString = paa.getCostPu() == 0 ? "" : "[" + paa.getCostPu() + " PU] ";
+  private String getActionButtonText(final PoliticalActionAttachment paa) {
+    final String costString = paa.getCostResources().isEmpty() ? ""
+        : "[" + ResourceCollection.toString(paa.getCostResources(), getData()) + "] ";
```

Comment:
Seeing this a few times:
```
ResourceCollection.toString(paa.getCostResources(), getData())
```

What do you think of converting it to this?
```
paa.getCostResourceString(getData())
```

Reply:
Generally the resource cost string is useful outside just political action attachments so fits better as a resource collection utility. Could add a helper method that then calls the utility method but probably not worth it in this case.

Reply:
The OO and trolling part of me wants to suggest: `new ResourceCostString(paa, getData())`    : )

Thanks, fair enough to leave as is IMO